monitor/alerts/model.py

```diff
@@ -1,33 +1,52 @@
 import json
-from dataclasses import dataclass
 from datetime import datetime
-from typing import Union
+from typing import List, Optional, Union
 
 from clients.slack.schema import SlackMessageSchema
 from monitor.alerts.alert import Alert
 from utils.json_utils import prettify_json_str_set
 
 
-@dataclass
 class ModelAlert(Alert):
-    unique_id: str
-    alias: str
-    path: str
-    materialization: str
-    detected_at: Union[str, datetime]
-    database_name: str
-    schema_name: str
-    message: str
-    full_refresh: bool
-    owners: str
-    tags: str
-    status: str
+    def __init__(
+        self,
+        id: str,
+        elementary_database_and_schema: str,
+        unique_id: str,
+        alias: str,
+        path: str,
+        materialization: str,
+        detected_at: Union[str, datetime],
+        database_name: str,
+        schema_name: str,
+        message: str,
+        full_refresh: bool,
+        owners: str,
+        tags: str,
+        status: str,
+        subscribers: Optional[List[str]] = None,
+        slack_channel: Optional[str] = None,
+        **kwargs
```

Comment:
Not sure why this kwargs is needed, let's talk?

Reply:
Alerts are generated from one place.
So we have one method that passes all the data every alert type needs (this is one of the things we talked about).
So it is much safer to add this **kwargs.